### DISCUSSION TITLE
Oqtane Developers Wanted

### DISCUSSION THREAD

**Discussion by TonyValenti** [oqtane/oqtane.framework#751] [Q&A]
[FasterLaw.com](http://www.FasterLaw.com) builds awesome .NET apps for legal professionals.

Our customers love us and we think we can do even better by leveraging .NET 5.0 and Oqtane.

We have a lot we'd like to do but we'd like to start with rebuilding:
1.  Our licensing portal
2.  Cloud-Based Time/Cost Recovery (our "Faster Time" app)

I am looking for a few **extremely** talented, work-remote individuals who want to be part of this project and are willing to commit between 15-40 hours per week.

If this sounds like you, email your resume' to [TV+OQTANE@FasterLaw.com](mailto:TV+OQTANE@FasterLaw.com) and we'll set up a time to talk further.

**Comment by hishamco** [comment]
I am not sure if this kind of request should be here

**Reply by mikecasas** [reply]
@TonyValenti My billable rate is $425 an hour. How many hours do you need?

**Reply by jimspillane** [reply]
@mikecasas is that in 15 or 20-minute minimum increments?

**Reply by mikecasas** [reply]
@jimspillane 15, I don't like being greedy ;)

**Comment by sbwalker** [comment]
FYI... I have known Tony Valenti for many years as his company was one of the most active members of the DNN ecosystem. This is a legitimate opportunity. If the community feels that the Discussions area is not the appropriate area for these types of requests, please make a suggestion on where members should ask for this type of assistance instead.

**Reply by kevind2000** [reply]
Is there a way to rename or create new category so that we could separate the technical help for internal projects and external project assistance, like this one?

**Reply by jimspillane** [reply]
I think the Discussions is a fine place for these requests.  Like @kevind2000 said, adding a Jobs category might be a good idea.

**Reply by markdav-is** [reply]
I was personally excited to see this here.  Discussions is our community forum, and we have categories, so thanks for posting!

**Comment by giffordwatkins** [comment]
> [FasterLaw.com](http://www.FasterLaw.com) builds awesome .NET apps for legal professionals.
> 
> Our customers love us and we think we can do even better by leveraging .NET 5.0 and Oqtane.
> 
> We have a lot we'd like to do but we'd like to start with rebuilding:
> 
> 1. Our licensing portal
> 2. Cloud-Based Time/Cost Recovery (our "Faster Time" app)
> 
> I am looking for a few **extremely** talented, work-remote individuals who want to be part of this project and are willing to commit between 15-40 hours per week.
> 
> If this sounds like you, email your resume' to [TV+OQTANE@FasterLaw.com](mailto:TV+OQTANE@FasterLaw.com) and we'll set up a time to talk further.

Tony... are you rebuilding your app on the Oqtane platform? (that might narrow the field a bit)

**Reply by TonyValenti** [reply]
@giffordwatkins - Yes, but not completely.  I want to Rebuild our leading apps taking into account things we've learned.

**Reply by markdav-is** [reply]
I still feel like I'm learning, but we are building an enterprise app as well as building out our oqtane talent pool.  So I'll keep you on my radar.  I might be a pro and not know it yet.

**Comment by PizzaConsole** [comment]
> I am looking for a few extremely talented

@TonyValenti  can you define "extremely talented" 
please?

**Reply by TonyValenti** [reply]
@PoisnFang - I'm looking for people who have a high degree of expertise with ASP.NET Core and specifically as much Oqtane experience as possible.

**Reply by hishamco** [reply]
@TonyValenti let me know how I can help you

**Reply by TonyValenti** [reply]
@hishamco Would you email TV+OQTANE@FasterLaw.com and we'll dialog that way?
